cmd/run_test.go

```diff
@@ -212,3 +213,83 @@ func TestRunScriptErrorsAndAbort(t *testing.T) {
 		})
 	}
 }
+
+// alarmist is a mock module that do a panic
+type alarmist struct {
+	vu modules.VU
+}
+
+var _ modules.Module = &alarmist{}
+
+func (a *alarmist) NewModuleInstance(vu modules.VU) modules.Instance {
+	return &alarmist{
+		vu: vu,
+	}
+}
+
+func (a *alarmist) Exports() modules.Exports {
+	return modules.Exports{
+		Named: map[string]interface{}{
+			"panic": a.panic,
+		},
+	}
+}
+
+func (a *alarmist) panic(s string) {
+	panic(s)
+}
+
+func TestRunScriptPanicsErrorsAndAbort(t *testing.T) {
+	t.Parallel()
+
+	modules.Register("k6/x/alarmist", new(alarmist))
```

Comment:
:question: to a reviwers, is there a way to do `Unregister`, should we implement something like that? :thinking: 

e.g. use case:
```golang
defer func (){
  modules.Register("k6/x/alarmist")
}()
```